common/reviews/api/streaming-image-volume-loader.api.md

```diff
@@ -1078,6 +1103,11 @@ interface IViewport {
     reset(immediate: boolean): void;
     setActors(actors: Array<ActorEntry>): void;
     setCamera(cameraInterface: ICamera, storeAsInitialCamera?: boolean): void;
+    setDisplayArea(

```

Comment:
When there are more than one option, the options should go into a typed options property so that the number of arguments doesn't change.